lib/utils/utf8.c

```diff
@@ -79,3 +79,46 @@ char *utf8_lcpy(char *dst, const char *src, size_t n)
 
 	return dst;
 }
+
+int utf8_count_chars(const char *s)
+{
+	ssize_t count = 0;
```

Comment:
Change the type to `int`

```suggestion
	int count = 0;
```